examples/sign.py

```diff
@@ -82,10 +84,8 @@ def callback_signer_es256(data: bytes) -> bytes:
                 "actions": [
                     {
                         "action": "c2pa.created",
-                        "parameters": {
-                            # could hold additional information about this step
-                            # eg. model used, etc.
-                        }
+                        "digitalSourceType": "http://cv.iptc.org/newscodes/digitalsourcetype/digitalCreation",
+                        "softwareAgent": "Python C2PA SDK example"
```

Comment:
If you want to add softwareAgent, it should be in the new format. It is no longer a String, its the same format as ClaimGeneratorInfo. Version and Icon are optional fields. But it may be simpler to just leave it out. It isn't required here.

"softwareAgent": {
    "name": "Python C2PA SDK example",
    "version": "0.1" 
  }